src/content/forms/heyform.md

```diff
@@ -0,0 +1,8 @@
+---
+path: "services/forms/heyform"
+title: "HeyForm"
+link: "https://heyform.net/"
+logo: "heyform.net"
```

Comment:
Looks like the URL is wrong here, should be `heyform.svg`

Reply:
Sorry about that Chris. I've made the edit. https://github.com/CSS-Tricks/serverless/pull/197/commits/ec4d8af8792a43c6aca97a1cbf5a8df64c881815
Please check the [preview](https://deploy-preview-197--vigorous-jennings-98a92a.netlify.app/).